src/pages/Account/AccountSecurity.tsx

```diff
@@ -1,11 +1,70 @@
 import * as React from 'react';
+import { ListGroup, ListGroupItem, Alert } from 'reactstrap';
+import { resetPassword } from 'utils/firebase';
+import { getDisplaySuccessMessage, SuccessMessage, getDisplayErrorMessage, ErrorMessage } from 'utils/validators';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import { User } from 'firebase';
 
-class AccountSecurity extends React.Component<any> {
-  render() {
-    return (
-      <h1>This is the Security Page</h1>
-    );
+interface AlertProperties {
+  color: string;
+  msg: string;
+}
+
+function AccountSecurity() {
+  const [alert, setAlert] = React.useState<AlertProperties>({ color: '', msg: '' });
```

Comment:
do the alerts just pop up in the space that you put it in? i wonder if we can just make it like the native browser alert

Reply:
discussed during standup. will use current behavior for now and consider building native browser alert component when we need it